src/main/kotlin/com/github/mdashl/rankedhelper/RankedHelper.kt

```diff
@@ -22,58 +15,60 @@ import net.minecraftforge.fml.common.Mod
 import net.minecraftforge.fml.common.event.FMLInitializationEvent
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent
 import org.jsoup.Jsoup
-import java.util.UUID
+import java.util.*
 import java.util.concurrent.Executors
 
-val thePlayer: EntityPlayerSP
-  get() = Minecraft.getMinecraft().thePlayer
-
-val pool = Executors.newFixedThreadPool(4).asCoroutineDispatcher()
+val playerSP: EntityPlayerSP
```

Comment:
Rename back to `thePlayer` as it better explains what is this and complicated with Forge's method.
```suggestion
val thePlayer: EntityPlayerSP
```